src/test/system_tests/sanity_build_test.js

```diff
@@ -4,114 +4,96 @@
 const _ = require('lodash');
 const api = require('../../api');
 const dbg = require('../../util/debug_module')(__filename);
-const ops = require('../utils/basic_server_ops');
 const argv = require('minimist')(process.argv);
-const blobops = require('../utils/blobops');
+
+if (process.env.SUPPRESS_LOGS) {
+    dbg.set_level(-5, 'core');
+}
 
 const { CloudFunction } = require('../utils/cloud_functions');
 const { BucketFunctions } = require('../utils/bucket_functions');
 
 const TEST_CTX = {
     client: null,
-    bucketfunc: null,
-    cloudfunc: null,
     aws_connection: null,
     azure_connection: null,
     compatible_v2: null,
     compatible_v4: null,
+    s3_endpoint: '',
+    mgmt_endpoint: '',
+    bucket_mirror: 'ec.no.quota',
+    bucket_spread: 'replica.with.quota',
+    ns_bucket: 'ns.over.azure.aws'
 };
 
-function show_usage() {
-    dbg.original_console();
-    console.info(`
-sanity_build_test.js
-Basic sanity test to be run as part of smoke testing. Configures the system, upgrades it and perform 2 simple I/O flows
-Usage:
-    --help                      Show this usage
-    --upgrade_pack <path>       Path to upgrade package
-    --target_ip <ip>            IP of server to test
-    \n
-    Examples:
-    \tnode src/test/system_tests/sanity_build_test.js --upgrade_pack ~/Downloads/noobaa-NVA-2.8.0-ede90e8.tar.gz --target_ip 52.151.33.226
-`);
-}
-
-function stop() {
-    process.exit(3);
-}
-
 async function main() {
-    if (argv.help) {
-        show_usage();
-        process.exit(0);
+    console.warn('NBNB:: ', process.env.TEST);
+    if (_.isUndefined(argv.mgmt_ip) || _.isUndefined(argv.mgmt_port)) {
+        console.error('Missing mgmt paramters');
+        process.exit(5);
     }
 
-    if (_.isUndefined(argv.upgrade_pack)) {
-        console.error('Missing upgrade_pack paramter!');
-        show_usage();
-        stop();
+    if (_.isUndefined(argv.s3_ip) || _.isUndefined(argv.s3_port)) {
+        console.error('Missing s3 endpoint paramters');
+        process.exit(5);
     }
 
-    if (_.isUndefined(argv.target_ip)) {
-        console.error('Missing target_ip paramter!');
-        show_usage();
-        stop();
+    if (_.isUndefined(process.env.AWS_ACCESS) || _.isUndefined(process.env.AWS_SECRET)) {
+        console.error('Missing env credentials');
+        process.exit(5);
     }
 
-    await run_test(argv.target_ip, argv.upgrade_pack, false);
+    TEST_CTX.s3_endpoint = `http://${argv.s3_ip}:${argv.s3_port}`;
```

Comment:
1. Need to keep in mind that if we build this endpoint here when we call it (line 96 for example) we should not write http:// again.
2. Building the endpoint needs to verify that the port is actually http and not https. 
3. The test_env_builder pass the ips and ports in the following format:
```
'--mgmt_ip', mgmt_ip,
'--mgmt_port', mgmt_port,
 '--mgmt_port_https', mgmt_port_https,
'--s3_ip', s3_ip,
'--s3_port', s3_port,
'--s3_port_https', s3_port_https,
```